Sources/MongoSwift/MongoError.swift

```diff
@@ -307,8 +307,7 @@ internal func convertingBulkWriteErrors<T>(_ body: () throws -> T) throws -> T {
 }
 
 internal func toErrorString(_ error: bson_error_t) -> String {
-    var e = error
-    return withUnsafeBytes(of: &e.message) { rawPtr -> String in
+    return withUnsafeBytes(of: error.message) { rawPtr in
```

Comment:
swift 4.2 introduced a non-inout version of this, so we no longer need to copy the error